spec/fixtures/state_file/fed_return_jsons/2023/nj/exempt_interest_over_10k.json

```diff
@@ -0,0 +1,46 @@
+{
+  "familyAndHousehold": [],
+  "filers": [
+    {
+      "firstName": "Ernie",
+      "middleInitial": null,
+      "lastName": "Muppet",
+      "dateOfBirth": "1980-01-01",
+      "isPrimaryFiler": true,
+      "tin": "400-00-0039"
+    },
+    {
+      "firstName": "Bert",
+      "middleInitial": "K",
+      "lastName": "Muppet",
+      "dateOfBirth": "1990-01-01",
+      "isPrimaryFiler": false,
+      "tin": "600-00-0039"
+    }
+  ],
+  "interestReports": [
+    {
+      "1099Amount": "0.00",
+      "has1099": false,
+      "interestOnGovernmentBonds": "5000.00",
+      "no1099Amount": "800.00",
+      "recipientTin": "400-00-0039",
+      "taxExemptInterest": "0.00",
+      "payer": "The payer name",
+      "payerTin": "101-23-4567",
+      "taxWithheld": "0.00",
+      "taxExemptAndTaxCreditBondCusipNo": "123456789"
+    },
+    {
+      "1099Amount": "0.00",
+      "has1099": false,
+      "interestOnGovernmentBonds": "5000.00",
```

Comment:
dust - it appears to me that there are two `interestOnGovernmentBonds` each equal to `5000.00` here, so a total of exactly 10k. In the ticket it says `greater than 10K`. Am I missing something, or should we add at least $1 to get this over 10k?

Reply:
ahhh ok nevermind, now I see the `TaxExemptInterestAmt` equal to 1 in the XML below, which would then be added to these to make `10,001`. I think I've answered my own question.

Reply:
I think I still have a side question about the paths for mocking via JSON vs XML, why we use some of each, and when we use each. But given there's a way these values combine it makes sense to me.

Reply:
JSON is the data collected by federal directfile, but not added to the federal return. For this one I explicitly wanted to make sure we tested that both values are represented, since that's necessary for 16b. I also wanted to make sure PMs had a way to check on the ineligible path/screen.